### ISSUE TITLE
Issue with "Reallife Parking?"

### ISSUE THREAD

**Issue by BlueSky-fur** [hoangducdt/MojiaGarages#58]
So, since the Update we installed the whole GarageSystem completly new.

now we get the issue that the "Reallife Parking" Update seems to be broken, when we buy cars the following error appears

```lua
[      script:oxmysql] SCRIPT ERROR in promise (unhandled): Error: qb-vehicleshop was unable to execute a query!
[      script:oxmysql]     Field 'posX' doesn't have a default value
[      script:oxmysql]     INSERT INTO player_vehicles (license, citizenid, vehicle, hash, mods, plate, state) VALUES ('license:d58604e18fe0791373e3538c85dc5ec1f66f8d7b', 'QLA32814', 'npolvette', -1109563416, '{}', '4MG573HZ', 0)
[      script:oxmysql] > query_default (@oxmysql/dist/server/build.js:16103)
[      script:oxmysql] > runMicrotasks (<anonymous>:0)
[      script:oxmysql] > processTicksAndRejections (node:internal/process/task_queues:96)
[      script:oxmysql] 
```

**Comment by BlueSky-fur** [comment]
It worked for only one single car
https://i.gyazo.com/9ef1289d01c6e6d863cdf5954edb9770.png

**Comment by mhnl3** [comment]
In the database add a default value of 0 for posX

**Comment by BlueSky-fur** [comment]
> In the database add a default value of 0 for posX

adressed and fixed in this PR 51ceee4d29c190a22f9050360f8645e9273b807d
